### ISSUE TITLE
Prism linux doesn't run in Apple Silicon

### ISSUE THREAD

**Issue by uesleilima** [Backbase/api-simulator#23]
Although the prism mac cli runs fine on Mac M1 laptops, the Prism Linux CLI doesn't run on Linux VMs using the ARM architecture.

Reference: https://github.com/stoplightio/prism/issues/2055

**Comment by uesleilima** [comment]
A workaround would be to set the platform when running the api-simulator API: `--platform=linux/amd64` to leverage the x86 architecture emulation (QEMU/Colima or Rosetta/Docker4Desktop)

**Comment by lehphyro** [comment]
I don't use a Mac so can't really help you, perhaps @ugurkaya88 can.

**Comment by uesleilima** [comment]
It is most likely to be fixed by the prism team first @lehphyro, I created this issue mainly for us to track their issue status and take action once we have a new version there.

**Comment by lehphyro** [comment]
Yeah, I figured that much, just wanted make you and @ugurkaya88 aware of the fact I won't be able to help (I'm usually the main maintainer of this project).
